README.md

```diff
@@ -175,7 +184,7 @@ Use https | `false` | `WLS_SECURE`
 
 ## Configure the exporter
 
-The sidecar does not have a landing page. Configuration is done by sending a PUT request to the path `/configuration`. 
+You configure the sidecar by sending a PUT request to the path `/configuration`. 
 You can do this with `curl`; when used with the [operator](https://github.com/oracle/weblogic-kubernetes-operator/), 
```

Comment:
The wording kind of implies the operator is using some sort of secret F&F API.  How about wording something like:

A sidecar is configured by sending an HTTP PUT request to the path `/configuration`; you can do this with `curl`.  

When a sidecar is used with WebLogic domains that are deployed using the [WebLogic Kubernetes Operator](https://github.com/oracle/weblogic-kubernetes-operator/), the WKO domain resource can be configured to automatically deploy and send configuration the sidecar.